packages/fxa-payments-server/src/App.tsx

```diff
@@ -48,6 +53,20 @@ export type AppProps = {
   routerOverride?: RouterOverride;
 };
 
+export function enableNavTiming(store: Store) {
+  let unsubscribe: ReturnType<typeof store.subscribe>;
+  const metricsObs = () => {
+    const profile = selectors.profile(store.getState());
+    // explicitly check for `false` because `undefined` means
+    // the user doesn't have an account or isn't signed in
+    if (profile?.result && profile.result.metricsEnabled !== false) {
+      observeNavigationTiming('/navigation-timing');
+      unsubscribe?.();
+    }
+  };
+  unsubscribe = store.subscribe(metricsObs);
+}
+
```

Comment:
Note the navigation timing metrics do not include any user specific data.  This still could be the right thing to do for an opt-out, but I want to make that clear.  😄

Reply:
👍 Yeah, it just seemed like the right thing to do considering the decision made in #7304. I think we might've needed a separate ticket out of that but we didn't make one for it so I just did it here. 😬